docs/topics/visibility-modifiers.md

```diff
@@ -57,16 +57,16 @@ For members declared inside a class:
 >
 {type="note"}
 
-If you override a `protected` member and do not specify the visibility explicitly, the overriding member will also have
-`protected` visibility.
+If you override a `protected` or a `private` member and do not specify the visibility explicitly, the overriding member
```

Comment:
You cannot override a `private` method. You can override `public`, `protected`, and `internal`.

Reply:
Sorry, mixed up the names. Fixed